packages/api/src/modules/auth/services.ts

```diff
@@ -15,34 +14,33 @@ import { LessThanOrEqual } from 'typeorm';
 
 export class AuthService implements IAuthService {
   async signIn(payload: ICreateUserTokenPayload): Promise<ISignInResponse> {
-    return UserToken.create(payload)
-      .save()
-      .then(data => {
-        return {
-          accessToken: data.token,
-          avatar: data.user.avatar,
-          address: data.user.address,
-          user_id: data.user.id,
-          expired_at: data.expired_at,
-          workspace: {
-            id: data.workspace.id,
-            name: data.workspace.name,
-            avatar: data.workspace.avatar,
-            single: data.workspace.single,
-            permissions: data.workspace.permissions,
-          },
-          ...(data.user.type === 'WEB_AUTHN'
-            ? { webAuthn: data.user.webauthn }
-            : {}),
-        };
-      })
-      .catch(e => {
-        throw new Internal({
-          type: ErrorTypes.Internal,
-          title: 'Error on token creation',
-          detail: e,
-        });
+    try {
+      const data = await UserToken.create(payload).save();
+
+      return {
+        accessToken: data.token,
+        avatar: data.user.avatar,
+        user_id: data.user.id,
+        expired_at: data.expired_at,
+        default_vault: data.user?.default_vault,
+        address: data.user?.address,
+        workspace: {
+          id: data.workspace.id,
+          name: data.workspace.name,
+          avatar: data.workspace.avatar,
+          single: data.workspace.single,
+          permissions: data.workspace.permissions,
+        },
+        first_login: data.user.first_login,
+        ...(data.user.type === 'WEB_AUTHN' ? { webAuthn: data.user.webauthn } : {}),
+      };
```

Comment:
Move this part to service, `UserToken.create().save()` and check if possible remove this line `...(data.user.type === 'WEB_AUTHN' ? { webAuthn: data.user.webauthn } : {})`

just keep table insert return